kvrocks.conf

```diff
@@ -597,6 +597,49 @@ rocksdb.enable_blob_garbage_collection yes
 # Default: 25
 rocksdb.blob_garbage_collection_age_cutoff 25
 
+# The size of all files per layer
+# consider setting this to a multiple (e.g. 8x or 10x) of target_file_size_base.
+#
+# Default 256 M
+rocksdb.max_bytes_for_level_base 256
+
+# Each subsequent level is max_bytes_for_level_multiplier larger than previous one. 
+# The default is 10 and RocksDB tunning guide do not recommend changing that unless you have special needs.
+#
+# Different max-size multipliers for different levels.
+# These are multiplied by max_bytes_for_level_multiplier to arrive
+# at the max-size of each level.
+#
+# Default 10
+rocksdb.max_bytes_for_level_multiplier 10
+
+
+
+# If true, RocksDB will pick target size of each level dynamically.
+# We will pick a base level b >= 1. L0 will be directly merged into level b,
+# instead of always into level 1. Level 1 to b-1 need to be empty.
+# We try to pick b and its target size so that
+# 1. target size is in the range of
+#   (max_bytes_for_level_base / max_bytes_for_level_multiplier,
+#    max_bytes_for_level_base]
+# 2. target size of the last level (level num_levels-1) equals to extra size
+#    of the level.
+# At the same time max_bytes_for_level_multiplier and
+# max_bytes_for_level_multiplier_additional are still satisfied.
+# (When L0 is too large, we make some adjustment. See below.)
+#
+# With this option on, from an empty DB, we make last level the base level,
+# which means merging L0 data into the last level, until it exceeds
+# max_bytes_for_level_base. And then we make the second last level to be
+# base level, to start to merge L0 data to second last level, with its
+# target size to be 1/max_bytes_for_level_multiplier of the last level's
+# extra size. After the data accumulates more so that we need to move the
+# base level to the third last one, and so on.
+#
+# Default False
+rocksdb.level_compaction_dynamic_level_bytes false
```

Comment:
Can we simplify this configuration item comment?
I think we should add some reference addresses (how about rocksdb wiki?). Instead of copying rocksdb's instructions, which are more detailed than that.

Reply:
> Can we simplify this configuration item comment? I think we should add a reference address(how about rocksdb wiki?) instead of copying rocksdb's comments, which are more detailed.

I've rewrited comments, thanks!